README.md

```diff
@@ -1,3 +1,72 @@
 # Comp330-Phase1
 Project for Comp 330
-Descirbe how to run the program. 
+
+-Descirbe how to run the program. 
```

Comment:
Inside Android Studio 3.4.1, run the Quiz.java file in the terminal. The questions should appear one, by one allowing the user to choose an answer. However, we were not able to get the code running.